conf/messages.cy

```diff
@@ -1259,12 +1259,13 @@ paymentAllocation.paymentAllocations.hmrcAdjustment.text        = Addasiad CThEF
 
 
 ## Tax Years Overview Update Tab ##
+updateTab.desc                                                  = Mae diweddariad chwarterol yn gyfanswm o incwm a threuliau ar gyfer y flwyddyn dreth hyd yn hyn.
```

Comment:
According to the Welsh doc this content should be :
Mae diweddariad chwarterol yn cynnwys incwm a threuliau ar gyfer y flwyddyn dreth hyd yn hyn

Reply:
Updated and verified - resolved